README.md

```diff
@@ -1,123 +1,177 @@
-## Nomad Rust implementations
+<img src=".github/images/Logo-White.svg" alt="Nomad logo" style="width: 100%; background: black;"/>
 
-### Setup
+## Nomad
 
-- install `rustup`
-  - [link here](https://rustup.rs/)
+Nomad is a cross-chain communication standard that supports passing messages between blockchains easily and inexpensively. Like [IBC](https://ibcprotocol.org) light clients and similar systems, Nomad establishes message-passing channels between chains. Once a channel is established, any application on that chain can use it to send messages to others chains.
 
-Note: You should be running >= version `1.52.1` of the rustc compiler, you can see that version with this command and should see similar output:
+Nomad is an implementation and extension of the [Optics protocol](https://medium.com/celoorg/announcing-optics-a-gas-efficient-interoperability-standard-for-cross-chain-communication-e597163b2) (**OPT**imistic **I**nterchain **C**ommunication), originally developed at Celo. 
 
+Compared to light clients, Nomad has weaker security guarantees and a longer latency period. However, these tradeoffs allow Nomad to be implemented on any smart contract chain without expensive light client development. Unlike light clients, Nomad does not use gas verifying remote chain block headers.
+
+Nomad is designed to prioritize:
```

Comment:
@prestwich do we want to say security here too? This would be in both README and docs site

Something like
```
Nomad is designed to prioritize:
...
- Security: relies on a 1/n honest watcher assumption for security
```

Reply:
@lattejed yeah we can add this bit about security above ^